src/feature/hs/hs_client.c

```diff
@@ -1261,12 +1471,12 @@ hs_client_note_connection_attempt_succeeded(const edge_connection_t *conn)
  * newly allocated descriptor object.
  *
  * Return 0 on success else a negative value and desc is set to NULL. */
```

Comment:
Let's update the function doc here, since while it's technically true, we now return a very detailed enum. The return value of this function is important since it floats up to the surface.

Reply:
Fixup commit 4194e6af10